### ISSUE TITLE
Final Fantasy X HD crashes without dumping shaders

### ISSUE THREAD

**Issue by ShadowsFriend** [doitsujin/dxvk#187]
While trying FFX HD, I experience crashes shortly after loading a game or starting a new one without a clear error. I tried enabling shader dumping for I thought this issue to be related with #166 but the specified folder is empty after the crash. Am I missing something or am I doing it wrong?

### Software information
Final Fantasy X HD on steam with various graphics settings (the last listed graphics pipeline in the logs seems to always be the same).

### System information
- GPU: NVIDIA GeForce GTX 660
- Driver: 387.34 (also tried 390.42 which is pretty bugged as can be seen in the [NVIDIA devtalk forum](https://devtalk.nvidia.com/default/topic/1029484/linux/-various-all-distros-numerous-performance-amp-rendering-issues-on-390-25/1)
- Wine version: 3.4 vanilla
- DXVK version: 42d49d7603401d662153b9fc4d94917f62c2bd92

### Log files
- d3d11.log: [FFX_d3d11.log](https://github.com/doitsujin/dxvk/files/1831303/FFX_d3d11.log)
- dxgi.log: [FFX_dxgi.log](https://github.com/doitsujin/dxvk/files/1831305/FFX_dxgi.log)

**Comment by ShadowsFriend** [comment]
Well, that's embarrassing. My toolchain seems to be at fault. I tried the precompiled libraries from https://haagch.frickel.club/files/dxvk/ and the game stopped crashing. Although I'm still not able to get shader dumps, I'm closing this thread for now.

**Comment by Glog78** [comment]
ShadowsFriend which version do you use ?

**Comment by ShadowsFriend** [comment]
What version are you interested in, @Glog78? I've listed the toolchain not working in https://github.com/doitsujin/dxvk/issues/184#issuecomment-375106832. And further down in that thread thirdeyefunction lists a working toolchain. In the meantime I've also figured out my mistake regarding the shader dumps. Dumping doesn't work in folders outside of specified wine drives. I'm always deleting the Z: drive pointing to root and tried to dump to /tmp/hb-dump which then didn't work. After specifying a path inside of the C: drive, dumping started to work.
If your question is about the FFX version, it's the most recent one from steam with a modified exe for Japanese voices (just some hardcoded paths hexedited, which shouldn't matter). The game works with wined3d albeit with choppy performance in some locations, which is why I decided to try dxvk.
